python/ray/llm/_internal/serve/deployments/llm/vllm/vllm_loggers.py

```diff
@@ -542,6 +542,9 @@ def record(
                 **lora_info_labels
             ).set_to_current_time()
 
+    def log_engine_initialized(self):
```

Comment:
Can you add a TODO here in this file that we should remove after the next release of vllm with xyz PR?